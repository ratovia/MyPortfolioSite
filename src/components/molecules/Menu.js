import React from "react"
import styled from "styled-components"
import { FrederickaH3 } from "../atoms/FrederickaText"
import { Link } from "gatsby"
const Flex = styled.ul`
  display: flex;
  flex-direction: row;
  justify-content: space-around;
  list-style: none;
  margin: 0;
  padding: 0;
  width: 100%;
  height: 50%;
  align-items: center;
`

const NavLink = styled(Link)`
  color: white;
  color: white;
  text-decoration: none;
`
export const Menu = () => {
  return (
    <Flex>
      <NavLink to="/">
        <li>
          <FrederickaH3 label="Top" />
        </li>
      </NavLink>
      <NavLink to="/about">
        <li>
          <FrederickaH3 label="About" />
        </li>
      </NavLink>
      <NavLink to="/work">
        <li>
          <FrederickaH3 label="Work" />
        </li>
      </NavLink>
      <NavLink to="/links">
        <li>
          <FrederickaH3 label="Links" />
        </li>
      </NavLink>
    </Flex>
  )
}
