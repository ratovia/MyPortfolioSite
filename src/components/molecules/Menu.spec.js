import React from "react"
import renderer from "react-test-renderer"
import { Menu } from "./Menu"
describe("Menu", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Menu />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
