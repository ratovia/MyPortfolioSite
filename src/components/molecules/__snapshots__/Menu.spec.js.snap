// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders correctly 1`] = `
<ul
  className="sc-eCImvq kJNpxb"
>
  <a
    className="sc-jRQAMF gvMAgo"
    href="/"
  >
    <li>
      <h3
        className="sc-hKwCoD NiySg"
      >
        <p
          className="sc-bdvvaa eMTeRp"
        >
          Top
        </p>
      </h3>
    </li>
  </a>
  <a
    className="sc-jRQAMF gvMAgo"
    href="/about"
  >
    <li>
      <h3
        className="sc-hKwCoD NiySg"
      >
        <p
          className="sc-bdvvaa eMTeRp"
        >
          About
        </p>
      </h3>
    </li>
  </a>
  <a
    className="sc-jRQAMF gvMAgo"
    href="/work"
  >
    <li>
      <h3
        className="sc-hKwCoD NiySg"
      >
        <p
          className="sc-bdvvaa eMTeRp"
        >
          Work
        </p>
      </h3>
    </li>
  </a>
  <a
    className="sc-jRQAMF gvMAgo"
    href="/links"
  >
    <li>
      <h3
        className="sc-hKwCoD NiySg"
      >
        <p
          className="sc-bdvvaa eMTeRp"
        >
          Links
        </p>
      </h3>
    </li>
  </a>
</ul>
`;
